vendor-overwrites/codemirror/addon/lint/css-lint.js

```diff
@@ -15,47 +15,85 @@
 })(function(CodeMirror) {
 "use strict";
 
-CodeMirror.registerHelper("lint", "css", function(text) {
-  var found = [];
-  if (!window.CSSLint) return found;
+CodeMirror.registerHelper("lint", "csslint", function(text) {
+  let found = [];
```

Comment:
ideally, it should be `const` as per our eslint rule, same for the occurrence below; however, this file has many other issues which you can see by removing vendor-overwrites/ from .eslintignore, so maybe you can fix them as well since you decided to reformat it but that's for you to decide.